### PR TITLE
feat: 通过环境变量, 禁用 copy 操作.

### PR DESCRIPTION
**Pull request by daya0576**
## PR 描述
通过环境变量, 禁用 copy 操作.

## 待办事项

* [x] 符合代码规范
* [x] 单元测试
* [x] 文档

### CODE REVIEW

File: tests/_test_env.py [hunk 1]
```diff
@@ -11,3 +13,15 @@
...
+def test_no_copy():
+    """ 禁用 copy 操作 """
+    reload(pypinyin.constants)
+    assert pypinyin.core.PINYIN_DICT is not pypinyin.pinyin_dict.pinyin_dict
+
+    os.environ['PYPINYIN_NO_DICT_COPY'] = 'true'
+    reload(pypinyin.constants)
+    assert pypinyin.constants.PINYIN_DICT is pypinyin.pinyin_dict.pinyin_dict
+
+
```
Comment (mozillazg) [inline tests/_test_env.py]: 这里多了个空行。

**Comment by mozillazg** [comment]
@daya0576 Thanks for your PR! 

Please fix the pep8 issue:

 - In the file [`tests/_test_env.py`](https://github.com/mozillazg/python-pinyin/blob/809b7d54b121e4421aae3c0927f14814b335ad96/tests/_test_env.py), following are the PEP8 issues :

> [Line 27:1](https://github.com/mozillazg/python-pinyin/blob/809b7d54b121e4421aae3c0927f14814b335ad96/tests/_test_env.py#L27): [W391](https://duckduckgo.com/?q=pep8%20W391) blank line at end of file

**Comment by daya0576** [comment]
Fixed. XD

How do u run travis-ci test locally? I tried to use docker but failed.

**Comment by daya0576** [comment]
Looks all good. Is there anything else should I do to improve this PR? 🤩

**Comment by mozillazg** [comment]
@daya0576 
> How do u run travis-ci test locally? I tried to use docker but failed.

Is there any error message?
You can use `make test` instead of `tox -e $TOX_ENV`, if you have problem about tox.

**Pull request merged**